ansible_collections/netapp/ontap/plugins/modules/na_ontap_command.py

```diff
@@ -206,10 +235,20 @@ def parse_xml_to_dict(self, xmldata):
                 self.result_dict['status'] = self.result_dict['xml_dict']['results']['attrs']['status']
                 stdout_string = self._format_escaped_data(self.result_dict['xml_dict']['cli-output']['data'])
                 self.result_dict['stdout'] = stdout_string
+                # Generate stdout_lines list
                 for line in stdout_string.split('\n'):
                     stripped_line = line.strip()
                     if len(stripped_line) > 1:
                         self.result_dict['stdout_lines'].append(stripped_line)
+
+                    # Generate stdout_lines_filter_list
+                    if self.exclude_lines:
+                        if len(stripped_line) > 1 and self.include_lines in stripped_line and self.exclude_lines not in stripped_line:
+                            self.result_dict['stdout_lines_filter'].append(stripped_line)
+                    else:
+                        if len(stripped_line) > 1 and self.include_lines in stripped_line:
```

Comment:
I wonder if there is a small performance in always testing that '' is in the line if the include_line filter is not used.

Also, is there any issue in returning twice the volume of output if no filter is defined?  self.result_dict['stdout_lines_filter'] will be an exact copy of self.result_dict['stdout_lines']

Reply:
@lonico - good catch. I've created a new PR - https://github.com/ansible/ansible_collections_netapp/pull/11